packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.controller.js

```diff
@@ -1,93 +1,100 @@
 import get from 'lodash/get';
-import orderBy from 'lodash/orderBy';
-
-import { PRODUCT_NAME } from './private-database-order-clouddb.constants';
+import {
+  pricingConstants,
+  workflowConstants,
+} from '@ovh-ux/manager-product-offers';
+import {
+  PRESELECTED_DB_CATEGORY,
+  PRODUCT_NAME,
+} from './private-database-order-clouddb.constants';
+import {
+  DATACENTER_CONFIGURATION_KEY,
+  DB_OFFERS,
+  ENGINE_CONFIGURATION_KEY,
+} from '../../../hosting/database/order/public/hosting-database-order-public.constants';
+import { DATABASES_TRACKING } from '../../../hosting/hosting.constants';
 
 export default class PrivateDatabaseOrderCloudDbCtrl {
   /* @ngInject */
-  constructor($filter, $translate, PrivateDatabaseOrderCloudDb) {
+  constructor(
+    $filter,
+    $translate,
+    PrivateDatabaseOrderCloudDb,
+    $anchorScroll,
+    atInternet,
+  ) {
     this.$filter = $filter;
+    this.$anchorScroll = $anchorScroll;
+    this.atInternet = atInternet;
     this.$translate = $translate;
     this.PrivateDatabaseOrderCloudDb = PrivateDatabaseOrderCloudDb;
   }
 
   $onInit() {
+    this.preselectDbCategory = PRESELECTED_DB_CATEGORY;
     this.currentIndex = 0;
     this.defaultModelType = PRODUCT_NAME;
-    this.model = {};
+    const { catalog, catalogItemTypeName } = this.getRightCatalogConfig(true);
+    this.productOffers = {
+      pricingType: pricingConstants.PRICING_CAPACITIES.RENEW,
+      workflowOptions: {
+        catalog,
+        catalogItemTypeName,
+        productName: this.getProductName.bind(this),
+        getPlanCode: this.getPlanCode.bind(this),
+        getRightCatalogConfig: this.getRightCatalogConfig.bind(this),
+        onGetConfiguration: this.getOnGetConfiguration.bind(this),
+      },
+      workflowType: workflowConstants.WORKFLOW_TYPES.ORDER,
+    };
+    this.model = {
+      dbCategory: {},
+    };
   }
 
-  selectTypeOption() {
-    this.model.type = this.defaultModelType;
-    this.currentIndex += 1;
+  trackClick(hit) {
+    this.atInternet.trackClick({
+      name: hit,
+      type: 'action',
+    });
   }
 
-  initializeCustomizationOptions() {
-    this.model.engine = undefined;
-    this.model.ramSize = undefined;
-
-    this.engineList = orderBy(
-      this.engines.map((engine) => {
-        const [engineLabel, engineVersion] = engine.split('_');
-
-        return {
-          engineLabel,
-          engineVersion: parseFloat(engineVersion),
-          label: this.$translate.instant(
-            `private_database_order_clouddb_server_version_${engineLabel}`,
-            {
-              version: engineVersion,
-            },
-          ),
-          value: engine,
-        };
-      }),
-      ['engineLabel', 'engineVersion'],
-      ['asc', 'desc'],
-    );
+  getOnGetConfiguration() {
```

Comment:
**nitpick:** `getOnGetConfiguration` name is confusing. Can you please rename the function?

Reply:
This naming comes from the historical public hosting database order controller.

Do we have to rename it there also to be consistent?

Reply:
renamed "getConfiguration"